### DISCUSSION TITLE
Offscreen & WPF Browser together

### DISCUSSION THREAD

**Discussion by bsod79** [cefsharp/CefSharp#4590] [General]
0

I need to use CEFSharp offscreen and standard Browser inside the same application... but I'm facing issue in initialization. I'm using something like that:

OffScreenBrowser = new biCefBrowser(true);
await OffScreenBrowser.InitializeBrowser(HTMLHome);

BiBrowser = new biCefBrowser(false);
await BiBrowser.InitializeBrowser(HTMLHome);
SupportWindow.RootGrid.Children.Add(BiBrowser.WpfBrowser);
But everything hangs.

I've already tried to split objects and force single CEF.Initialize

**Comment by amaitland** [comment]
Add the `WPF` browser to the grid **before** you try and initialize it. The default is to only create the underlying browser when the control gets an actual size.
